pydicom/config.py

```diff
@@ -35,7 +41,12 @@ def __call__(
 """
 
 
-data_element_callback: Optional["ElementCallback"] = None
+data_element_callback: Optional[
+    Union["ElementCallback", List["ElementCallback"]]] = [
+    pydicom_raw_elem_convert_value,
+    pydicom_raw_elem_preprocess,
+    pydicom_raw_elem_postprocess
+]
 """Set to a callable function to be called from
```

Comment:
I don't think these pydicom callbacks should be here - if the user code replaces them, (as existing codes have done) then it is a breaking change -  they will not be called at all.  I'm not sure how to fix it - they should normally be called in the conversion routines, but I imagine you may want to replace them.  Perhaps one or more new config callback types are necessary?  Will have to mull this over.